glotaran/test/test_spectral_decay_full_model.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
+import pytest
+
+from glotaran.analysis.optimize import optimize
+from glotaran.analysis.simulation import simulate
+from glotaran.io import load_model
+from glotaran.io import load_parameters
+from glotaran.project import Scheme
+
+MODEL_3C_BASE = """\
+dataset:
+    dataset1: &dataset1
+        megacomplex: [mc1]
+        global_megacomplex: [mc2]
+        initial_concentration: j1
+        irf: irf1
+    dataset2: &dataset2
+        megacomplex: [mc2]
+        global_megacomplex: [mc1]
+        initial_concentration: j1
+        irf: irf1
+    dataset3: &dataset3
+        megacomplex: [mc1]
+        initial_concentration: j1
+        irf: irf1
+    dataset4: &dataset4
```

Comment:
```suggestion
    dataset1:
        megacomplex: [mc1]
        global_megacomplex: [mc2]
        initial_concentration: j1
        irf: irf1
    dataset2:
        megacomplex: [mc2]
        global_megacomplex: [mc1]
        initial_concentration: j1
        irf: irf1
    dataset3:
        megacomplex: [mc1]
        initial_concentration: j1
        irf: irf1
    dataset4:
```
The anchors aren't used, so there is no need to define them.